buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/SpinePublishing.kt

```diff
@@ -242,6 +242,28 @@ open class SpinePublishing(private val project: Project) {
      */
     fun testJar(configuration: TestJar.() -> Unit)  = testJar.run(configuration)
 
+
+    /**
+     * Configures publishing of [dokkaJar] artifact, containing Dokka-generated documentation. By
+     * default, this option is set to `false`.
+     *
+     * Remember that the Dokka Gradle plugin should be applied to publish this artifact as it is
+     * produced by the `dokkaHtml` task. It can be done by using the
+     * [io.spine.internal.dependency.Dokka] dependency object or by applying the
+     * `buildSrc/src/main/kotlin/dokka-for-java` script plugin for Java projects.
+     *
+     * Here's an example of how to use this option:
+     *
+     * ```
+     * spinePublishing {
+     *     includeDokkaJar = true
+     * }
+     * ```
+     *
+     * The resulting artifact is available under "dokka" classifier.
+     */
+    var includeDokkaJar = false
```

Comment:
Let's make this expression syntactically similar to `protoJar` (see above). 

Also, the naming is important. We drop "include" prefix in order to make the whole thing shorter. Imagine if we don't:
```
spinePublishing {
    // ...

    
    includeTestJar { ... }
    includeProtoJar { ... }
    includeDokkaJar ...
}
```
Notice how much space this "include" takes? And the person who reads this needs to traverse down to the meaningful part, skipping "include" every time.